### DISCUSSION TITLE
Adicionar estilização

### DISCUSSION THREAD

**Discussion by jersonb** [Projeto-Faculdade/projeto-facul#37] [General]
> - Como desenvolvedor

> - Desejo entregar software com qualidade de mercado

> - Para aumentar a qualidade do software

**Comment by jersonb** [comment]
@a-mandaflores você tem alguma opinião a respeito de frameworks de estilização para front-end

**Comment by a-mandaflores** [comment]
Podemos utilizar o Bootstrap. Acredito que a maioria ja tem familiaridade, é fácil de trabalhar e fica legal no final das contas. O que acha?

**Comment by a-mandaflores** [comment]
## Design de baixa fidelidade

![Image](https://github.com/Projeto-Faculdade/projeto-facul/assets/61766435/3235963d-b762-4e5b-adf0-32eb8e20fcd7)



![Image](https://github.com/Projeto-Faculdade/projeto-facul/assets/61766435/ab53339d-92ab-46c2-bec9-c7cd4a7b215e)

## Task
- Não vamos utilizar o framework pelo peso que ele vai dar no site depois.

- Vamos utilizar o Google Shets como nosso Banco de Dados

- Precisamos fazer a configuração para fazer um crud com o Google Shets (DESAFIO)

- A atualização da atividades vai ser como Single page aplication(DESAFIO)
